troupon/merchant/views.py

```diff
@@ -139,7 +139,9 @@ def post(self, request, **kwargs):
         active = True if active else False
         image = request.FILES.get('image')
 
-        date_end_unicode = request.POST.get('date_end')
+        date_request = request.POST.get('date_end')
```

Comment:
**Code Quality: Nitpick**

Two possibilities here. 

A. You could replace this with a single line

```py
date_end_unicode = request.POST.get('date_end') or (date.today() + timedelta(days=2)).isoformat()
```

B. You could replace this with a single line :smile: 

```py
date_end_unicode = request.POST.get('date_end', (date.today() + timedelta(days=2)).isoformat())
```